sys/README.md

```diff
@@ -55,6 +55,8 @@ rest of the type-options are type-specific:
 	"bytesize": similar to "len", but always denotes the size in bytes, type-options:
 		argname of the object
 	"vma": a pointer to a set of pages (used as input for mmap/munmap/mremap/madvise)
+	"proc": per process int, type-options:
```

Comment:
Please extend this description. It is very non-obvious type. I doubt that anybody expect me and you understand what it does.
A usage example may help.